connect/runtime/src/main/java/org/apache/kafka/connect/runtime/WorkerSinkTask.java

```diff
@@ -193,13 +194,11 @@ public void transitionTo(TargetState state) {
     @Override
     public void execute() {
         initializeAndStart();
-        try {
+        // Make sure any uncommitted data has been committed and the task has
+        // a chance to clean up its state
+        try (UncheckedCloseable supressible = this::closePartitions) {
```

Comment:
Although I dig supressible with 1 p, like "sup? exception what are you up to?" 
I'm afraid I'll have to abide by the usual rules and recommend: `suppressible` here 😄 
(probably the IDE will complain about a typo too).

Reply:
```suggestion
        try (UncheckedCloseable suppressible = this::closePartitions) {
```